03-Control-Attestation/README.md

```diff
@@ -0,0 +1,337 @@
+- [How scanner determines the effective control result](README.md#how-scanner-determines-the-effective-control-result)  
+- [Permissions required for attesting controls](README.md#permissions-required-for-attesting-controls) 
+- [Attestation expiry](README.md#attestation-expiry)  
+
+----------------------------------------------
+
+## Overview
+
+The attestation feature empowers users to support scenarios where human input is required to augment or override the default control 
+evaluation status from AzSK.ADO. These may be situations such as:
+
+- The scanner has generated the list of 'Administrators' for a project but someone needs to have a look at the list and ratify that 
+these are indeed the correct people, or
+- The scanner has marked a control as failed. However, given the additional contextual knowledge, the application owner wants to ignore the control failure.
+
+In all such situations, there is usually a control result that is based on the technical evaluation (e.g., Verify, Failed, etc.) that has to 
+be combined with the user's input in order to determine the overall or effective control result. The user is said to have 'attested' such controls 
+and, after the process is performed once, AzSK.ADO remembers it and generates an effective control result for subsequent control scans _until_ there 
+is a state change.
+
+The attestation feature is implemented via a new switch called *ControlsToAttest* which can be specified in any of the standard security scan cmdlets
+of AzSK.ADO. When this switch is specified, the scanner first performs a scan of the target resource(s) like it is business as usual and, once
+the scan is complete, it enters a special interactive 'attest mode' where it walks through each resource and relevant attestable controls
+and captures inputs from the user and records them (along with details about the person who attested, the time, etc.). 
+After this, for all future scans on the resource(s), AzSK.ADO will show the effective control evaluation results. Various options are provided to support
+different attestation scenarios (e.g., expiry of attestations, edit/change/delete previous attestations, attest only a subset of controls, etc.). 
+These are described below. Also, for 'stateful' controls (e.g., "are these the right service accounts with access to project collection?"), the attestation
+state is auto-reset if there is any change in 'state' (e.g., someone adds a new service account to the list).
+
+Lastly, due to the governance implications, the ability to attest controls is available to a subset of users. This is described in
+the permissions required section below.  
+
+[Back to top...](README.md#control-attestation)
+
+----------------------------------------------
+
+## Setup Attestation Repository
+
+AzSK.ADO internally stores attestation details in a project repository named 'ADOScannerAttestation' which needs to be configured typically by the project admin. 
```

Comment:
'ADOScannerAttestation' is default name for attestation repo, Repo name and branch name also can be configured from org-policy